clean-app.sh

```diff
@@ -8,27 +8,32 @@ delete_data() {
 	if defaults read "${bundle_id}" &>/dev/null; then
 		defaults delete "${bundle_id}"
 	fi
-	rm -rf "${HOME}/Library/Containers/${bundle_id}/Data"
+	rm -r "${HOME}/Library/Containers/${bundle_id}/Data"
```

Comment:
Why are we removing `-f`? This will break the script when the directory doesn't exist.

Reply:
well, it does for me; 
but it does report a missing dir status instead if you run it twice

Reply:
Sorry about that, the script actually goes on (we’re not using `set -e` here, but displays an error). Would be great to have it handled gracefully if we’re skipping `-f`. Thanks for updating it.